misc/config/config.nydus.ref.yaml

```diff
@@ -64,4 +64,4 @@ converter:
     # add suffix to tag of source image reference as target image reference
     - tag_suffix: -nydus-oci-ref
     # add suffix to tag of source image reference as remote cache reference, leave empty to disable remote cache.
```

Comment:
`set tag of source image reference as remote cache reference, leave empty to disable remote cache.` :)